src/features/vote/fetchUMIP.ts

```diff
@@ -0,0 +1,27 @@
+import * as contentful from "contentful";
+
+const contenfulSpaceId = process.env.REACT_APP_PUBLIC_CONTENTFUL_SPACE_ID;
+const contentfulAccessToken =
+  process.env.REACT_APP_PUBLIC_CONTENTFUL_ACCESS_TOKEN;
+
+const contentfulClient = contentful.createClient({
+  space: contenfulSpaceId || "",
+  accessToken: contentfulAccessToken || "",
+});
+
+export type UMIP = {
+  description: any;
+  discourseLink?: string;
+  status?: string;
+  authors?: string;
+  title: string;
+  number: number;
+};
+
+export const fetchUmip = async (number: number) => {
+  const ct = await contentfulClient.getEntries<UMIP>({
+    content_type: "umip",
+    "fields.number": number,
+  });
+  return ct.items[0].fields;
+};
```

Comment:
nice moving this out into just a regular ts file.  it would be nice to wrap this in a function or class so that code isnt just executed when the file is imported, but its probably not a big deal for now.

Reply:
I have built apiService type logic before as an object -- could always repeat.

Reply:
up to you, not critical at this point